src/System.Diagnostics.Process/tests/ProcessThreadTests.Unix.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public partial class ProcessThreadTests
+    {
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [Fact]
+        public void TestPriorityLevelProperty_Unix()
+        {
+            CreateDefaultProcess();
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                Assert.Throws<PlatformNotSupportedException>(() => _process.Threads[0].PriorityLevel);
```

Comment:
Why is the getter only tested on OSX and not on Linux?

Reply:
That's a mistake.  Fixing.